waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -1,5 +1,202 @@
+  justify-content: start;
+`;
+
+const Logo = styled.img.attrs({
+  src: `${LogoSVG}`,
+  alt: 'logo',
+})`
+  width: 260px;
+  height: 150px;
+  margin-bottom: -15px; // required to make up for slight gap in the svg
+`;
+
+const ContactDiv = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: start;
+`;
+
+const ContactUs = styled.p`
+  font-size: 18px;
+  margin: 10px 0;
+  color: ${fill};
+  font-weight: bold;
+`;
+
+const ContactInfo = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+`;
+
+const EmailIcon = styled.img.attrs({
+  src: `${EmailSVG}`,
+  alt: 'email',
+})`
+  width: 28px;
+  height: 24px;
+`;
+
+const ContactEmail = styled.p`
+  font-size: 14px;
+  margin: 0 5px;
+  color: ${fill};
+`;
+
+const Divider = styled.hr`
+  width: 100%;
+  border-color: ${fill};
+  margin: 10px 0;
+  border-width: 1px;
+`;
+
+const PageTagsAndSocialsWrapper = styled.div`
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  @media screen and (max-width: 990px) {
+    flex-direction: column;
+  }
+`;
+
+const PageTagsWrapper = styled.div`
+  //   width: 30vw;
+  display: flex;
+  flex-direction: row;
+`;
+
+const Tag = styled.p`
+  font-size: 16px;
+  color: ${fill};
+  margin-right: 50px;
+  cursor: pointer;
+`;
+
+const Tags = [
+  // tags are dummy for now as I'm not aware of what tags we'll be using for each section.
+  { text: 'Home', tag: 'dummy' },
+  { text: 'About', tag: 'dummy' },
+  { text: 'Schedule', tag: 'dummy' },
+  { text: 'Team', tag: 'dummy' },
+  { text: 'Sponsors', tag: 'dummy' },
+  { text: 'FAQ', tag: 'dummy' },
+];
+
+const Socials = [
+  { SVG: TwitterSVG, link: 'https://twitter.com/team_waterloop' },
+  { SVG: InstagramSVG, link: 'https://www.instagram.com/team_waterloop/' },
+  { SVG: FacebookSVG, link: 'https://www.facebook.com/teamwaterloop' },
+  { SVG: LinkedInSVG, link: 'https://www.linkedin.com/company/wloop' },
+];
+
+const SocialsWrapper = styled.div`
+  display: flex;
+`;
+
+const SocialBubble = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 50px;
+  height: 50px;
+  background-color: #fff;
+  border-radius: 50%;
+  margin-left: 25px;
+`;
+
+interface SocialProps {
+  SVG: string;
+}
+
+const SocialImg = styled.img.attrs((props: SocialProps) => {
+  return {
+    src: `${props.SVG}`,
+  };
+})<SocialProps>`
+  width: 14px;
+  height: 14px;
+`;
+
+const FooterImg = styled.img.attrs({
+  src: STEFooterImg,
+})`
+  width: 100%;
+  height: 100%;
+  margin-bottom: -0.5rem;
+  background-color: #d1e6f5;
+`;
+
+const STEFooter: React.FC = () => (
+  <>
+    <FooterImg />
+    <FooterBackground>
+      <LogoAndContactWrapper>
+        <Logo />
+        <ContactDiv>
+          <ContactUs>Contact Us</ContactUs>
+          <ContactInfo>
+            <EmailIcon /> <ContactEmail>contact@waterloop.ca</ContactEmail>
```

Comment:
The formatting of this seems a bit off, compared to the Figma design (margin + alignment)

Reply:
Adjustments made, will push in upcoming commit.
![image](https://user-images.githubusercontent.com/71536798/156801782-f55d7d24-d670-488d-abbc-137e598e18d8.png)